Dockerfile

```diff
@@ -14,14 +14,21 @@ RUN apt-get update && apt-get install -y \
 	&& rm -rf /var/lib/apt/lists/*
 
 RUN useradd -ms /bin/bash notary \
-	&& pip install codecov \
-	&& go get golang.org/x/lint/golint github.com/fzipp/gocyclo github.com/client9/misspell/cmd/misspell github.com/gordonklaus/ineffassign github.com/securego/gosec/cmd/gosec/...
+    && pip install codecov
 
-ENV NOTARYDIR /go/src/github.com/theupdateframework/notary
+ENV GO111MODULE=on
+
+# Locked go cyclo on this commit as newer commits depend on Golang 1.16 io/fs
+RUN go get golang.org/x/lint/golint \
+    github.com/client9/misspell/cmd/misspell \
+    github.com/gordonklaus/ineffassign \
+    github.com/securego/gosec/cmd/gosec/... \
+    github.com/fzipp/gocyclo@0c41b8201a7b6d814898607c4e0cd097c0f2ae46
```

Comment:
What about using this commit [`ffe36aa317dcbb421a536de071660261136174dd`](https://github.com/fzipp/gocyclo/tree/ffe36aa317dcbb421a536de071660261136174dd)? It's from April 8th, 2021, which  is newer than `0c41b8201a7b6d814898607c4e0cd097c0f2ae46` which is from October 16th, 2020, and the version in `go.mod` is still `1.15`.

Reply:
Good point. No clue how I ended up on that commit. ffe36aa317dcbb421a536de071660261136174dd this is much better.